src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -186,10 +189,32 @@
           + "Values can be `PLAINTEXT` or `SSL`. If `PLAINTEXT` is passed, "
           + "all configs prefixed by " + CONNECTION_SSL_CONFIG_PREFIX + " will be ignored.";
 
+  // Proxy group
+  public static final String PROXY_HOST_CONFIG = "proxy.host";
+  public static final String PROXY_HOST_DISPLAY = "Proxy Host";
+  public static final String PROXY_HOST_DOC = "The address of the proxy host to connect through.";
```

Comment:
Can we be more explicit about which proxies are and aren't supported?  I remember we had an issue where it was unclear if we supported NTLM proxy.

Reply:
addressed